readme/simplesamlphp-setup.md

```diff
@@ -115,6 +115,16 @@ Request the remote IdP metadata (XML) from the customer. Note that each environm
 
 1. Commit your changes to your Git repository.
 
+## Testing Library Configuration
+
+Once you have completed the code changes above, you should proceed with testing the configuration of SimpleSAMLphp as a Service Provider connecting to your specified Identity Provider.
+
+You can do this by navigating to Go to the path `/simplesaml/module.php/core/authenticate.php` on your application to access the SimpleSAMLphp library Test authentication sources page. On this page you will need to click the authsource to test. It should be default-sp unless configured otherwise.
```

Comment:
@guitarte this sentence seems wonky 'You can do this by navigating to Go to the path'

Should it just be 'You can do this by navigating to'?

Reply:
Totally messed up on a copy/paste - fixed in my latest commit 32b87e2